ui/raidboss/data/03-hw/dungeon/the_vault.js

```diff
@@ -0,0 +1,292 @@
+    },
+    {
+      id: 'The Vault Heavy Swing',
+      regex: /Heavy Swing/,
+      beforeSeconds: 4,
+      infoText: function(data) {
+        if (data.role == 'tank') {
+          return {
+            en: 'Tank cleave',
+          };
+        }
+        return {
+          en: 'Avoid tank cleave',
+        };
+      },
+    },
+    {
+      id: 'The Vault Altar Candle',
+      regex: /Altar Candle/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        return data.role != 'dps';
+      },
+      alertText: {
+        en: 'Tank buster',
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'The Vault Holiest of Holy',
+      regex: / 14:101E:Ser Adelphel starts using Holiest Of Holy/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'The Vault Holy Shield Bash',
+      regex: /14:101F:Ser Adelphel starts using Holy Shield Bash on (\y{Name})/,
+      condition: function(data, matches) {
+        return data.me != matches[1];
+      },
+      alertText: function(data, matches) {
+        if (data.role == 'healer') {
+          return {
+            en: 'Heal + shield ' + data.shortName(matches[1]),
+          };
+        }
+        return {
+          en: 'Avoid Shield Charge',
+        };
+      },
+    },
+    {
+      id: 'The Vault Execution',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:0020/,
+      infoText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Spread marker on YOU',
+          };
+        }
+        return {
+          en: 'Avoid ' + data.shortName(matches[1]),
+        };
+      },
+    },
+    {
+      id: 'The Vault Black Nebula',
+      regex: / 14:1042:Face Of The Hero starts using Black Nebula/,
+      condition: function(data) {
+        return data.CanStun();
+      },
+      infoText: {
+        en: 'Interrupt the Knight',
+      },
+    },
+    {
+      id: 'The Vault Faith Unmoving',
+      regex: /14:1027:Ser Grinnaux starts using Faith Unmoving/,
+      infoText: {
+        en: 'Knockback',
+      },
+    },
+    {
+      id: 'The Vault Dimensional Torsion',
+      regex: /23:\y{ObjectId}:Aetherial Tear:\y{ObjectId}:(\y{Name}):....:....:0001/,
+      suppressSeconds: 5,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      alarmText: {
+        en: 'Away from rifts',
+      },
+    },
+    {
+      id: 'The Vault Altar Pyre',
+      regex: / 14:1035:Ser Charibert starts using Altar Pyre/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      alertText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'The Vault Holy Chains',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:0061/,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      alertText: {
+        en: 'Away from tether partner',
```

Comment:
I wasn't sure on this one myself. This particular tether doesn't have the visual indicators like some do to inform the player how to resolve it, so I thought this would be a short way to convey *how* to break it. But if it's better to just tell the player *what* is happening, that's fine too.

Reply:
I guess I think most tethers are "move away from tether partner" except for maybe like one of them? What sort of visual indicator other than the line connecting you are you thinking about?

Reply:
I think the best example of the visual is the light purple tethers in Omega 6. It looks like a double-ended conveyor belt of pyramids moving away from the central point of the tether to the players. "Move the ends of the tether THAT way". But you're right, now that I think about it the only major collapse tether is in Ifrit EX.